docs/src/modules/javascript/pages/value-entity-videos.adoc

```diff
@@ -0,0 +1,20 @@
+= Value entities and stateful services: Video 1-2-3
+
+The following videos will give you a quick overview of building a start service with value entities.
+
+First, a review of what Akka Serverless provides for your developers.
+
+link:https://www.youtube.com/watch?v=D9_JM2sRh98[Akka Serverless - the best of serverless and state without any hassle, role=yt-widget]
+
+Now, just view the three following videos. In less than five minutes you will cover the basics.
```

Comment:
I'd recap a part of the `Simplified Programming Model` here, like

"In less than five minutes, you can build a stateful serverless service using Value Entities. These videos walk you through the three steps to you need to develop:
1. Define the API and the domain data model in gRPC descriptors.
2. Choose a state model (this is your Value Entity).
3. Implement your business logic in JavaScript."

Reply:
cool - adding now